ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -259,6 +259,25 @@ post_provision_config_nodes() {
 
     lsb_release -a
 
+    if [ -n "$INST_REPOS" ]; then
+        local repo
+        for repo in $INST_REPOS; do
+            branch="master"
+            build_number="lastSuccessfulBuild"
+            if [[ $repo = *@* ]]; then
+                branch="${repo#*@}"
+                repo="${repo%@*}"
+                if [[ $branch = *:* ]]; then
+                    build_number="${branch#*:}"
+                    branch="${branch%:*}"
+                fi
+            fi
+            local repo_url="${JENKINS_URL}"job/daos-stack/job/"${repo}"/job/"${branch//\//%252F}"/"${build_number}"/artifact/artifacts/$DISTRO_NAME/
+            dnf -y config-manager --add-repo="${repo_url}"
+            disable_gpg_check "$repo_url"
+        done
+    fi
+
```

Comment:
I'm not sure we want to land this without some more thought on potential fall-out and testing.